src/app/attendance/actions/attendanceActions.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { describe, it, vi, expect } from "vitest";
```

Comment:
File name is not correct. we're over contextualizing. Since this is already under actions directory, it can simply be called `createAttendance`

Reply:
Fix